source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -276,6 +283,12 @@ def _get_mode_weight(journey):
     return max(mode_weight.get(mode, 1) for mode in journey.tags)
 
 
+def get_min_waiting(journey):
+    """
+    Returns min waiting time in a journey
+    """
+    return min([s.duration for s in journey.sections if s.type == response_pb2.WAITING] or [0])
```

Comment:
finally, no, as we use this function with same number of transfer. So, if there is a waiting in one and no waiting in the other, that's because the waiting is 0.